routes/api/queries/tula/Network.js

```
@@ -214,6 +215,114 @@ function getLevelStyles(network) {
   }));
 }
 
+/**
+ * Return a string containing a set of records suitable for SQL insertion.
+ * @param  {Object} network - the ATLAS compliant network
+ * @return {Array<Object>}  - an array containing SQL field tuples
+ */
+function getValueClause(network) {
+   let ValueClauses = [];
+   for (let i = 0; i < network.stats.length; i++) {
+     let name = kawokTranslation[i].atlasProperties.name;
+     ValueClauses.push({nodeId: network.id, indicName: name, indicValue: network.stats[i]});
+   }
+   return ValueClauses.concat(...network.children.map(c => getValueClause(c)));
+ }
+
+/**
+ * Store fetched data in PSQL table with timestamps for later usage in
+ * building timelines. Only add new rows if indicator values have changed.
+ * @param  {Object} data  - data structure returned by fetchData.
+ */
+async function storeData(data) {
+  debugger;
+  try {
+    await db.query(SQL`BEGIN`);
+  } catch (err) {
+    console.log(err);
```

We shouldn't be console logging this much.  If there is nothing you need to do to handle the error, just fail silently, or have a single console log to notify that there was a failure storing the timeline data.

---

Done.